nuxeo-features/nuxeo-platform-audit/nuxeo-platform-audit-core/src/main/resources/directories/event-types.csv

```diff
@@ -2,6 +2,7 @@ id,label,obsolete,ordering
 "documentCreated","documentCreated","0","0"
 "documentSecurityUpdated","documentSecurityUpdated","0","10"
 "documentModified","documentModified","0","30"
+"documentRemoved","documentTrashed","0","35"
```

Comment:
Shouldn't be `"documentTrashed","documentRemoved"`? (id,label)
Will we need `documentUntrashed` ?

Reply:
yeah sorry, I created this PR by mistake